### PR TITLE
Standardized tooltip style in app; Navbar changes; workspace revamp

### PR DESCRIPTION
**Pull request by nattiechan**
- Create a separation between the core and experimental categories so users have a better expectation of how robust a
feature should be

https://user-images.githubusercontent.com/47332368/230747059-76a7c408-ee15-43f0-96d9-9a4949678b8f.mov

- Fix tooltip not showing up for load test: switching to using Material UI's tooltip instead. Adding additional tooltip prompt reminding users that the test corresponds to the workspace pane to the left

https://user-images.githubusercontent.com/47332368/230747078-80255634-53c8-4957-84a0-a6d6d4dc0700.mov

- Standardized tooltip style in the app. As a result, the "current workspace" management (i.e. save, select, delete) is overhauled. Updated state management throughout the current workspace workflow to reduce prop drilling

- The scheduled feature no longer exists (and replaced by load testing), so we removed any code associated with the obsolete feature

### CODE REVIEW

**Comment by nattiechan** [comment]
@Fredkim21 @jjhuang3 @csuzukida I would really appreciate some eyes on this and actually testing the functionality locally. Do:

`git checkout dev && git pull && git checkout navbar`

Which will pull this `navbar` branch from remote into your local machine and switch to said branch. Once you are in the branch, please run `npm run dev` and test the HTTP/2 and GraphQL load testing features. I have a tough time finding HTTP endpoints that would not give me CORS issues (maybe @Fredkim21 knows better?) So I really need some help testing that

File: src/client/components/workspace/TestContainer.tsx [hunk 1]
```diff
@@ -68,19 +49,29 @@ const TestContainer: React.FC<TestContainerProps> = ({
     : null;
 
   const getDisabledReason = () => {
+    const basePrompt = `
+      Please note that this load test will execute 
+      the selected request in the workspace to the left.
+    `;
```
Comment (nattiechan) [inline src/client/components/workspace/TestContainer.tsx:55]: @Fredkim21 fixed it here. What do you think?

Comment (Fredkim21) [reply src/client/components/workspace/TestContainer.tsx:55]: yeah looks good

**Review by Fredkim21 — APPROVED** [review]
looks good.

**Pull request merged**